docs/release_notes/mar2023.md

```diff
@@ -8,6 +8,49 @@ tags:
   - changelog
 ---
 
+## 2.10.0.0 (Mar 31, 2023)
+
+- Prophecy Python libs version: 1.4.8
+- Prophecy Scala libs version: 7.0.16
+
+### Updates {#UpdatesRelease2100}
+
+#### Support for Personal Access Tokens
```

Comment:
Was the API accessible previously without PATs? Is this a security improvement? Would this be correct?
"The Prophecy metadata API is now accessible only via Personal Access Token"

Perhaps we want to "Introduce" the metadata API as though it's the first time this API is accessible, and it's a very secure approach, like this: 
"Now Prophecy users can access their metadata using the Prophecy API. Just create a Personal Access Token (PAT) (link). One popular use case is to access the Prophecy metadata through automated scripts and integrate this data into an existing data visualization system. Use the Prophecy API to create a single pane of glass interface for data stewards and engineers to see pipelines, datasets, Lineage, etc. Please see [here] for more details on this feature."